process-rbac-foo.sh

```diff
@@ -0,0 +1,2 @@
+
+bonfire process rbac -n foo
```

Comment:
Was this file accidentally included?

Reply:
it was, sorry - I use this script locally for some sort of E2E testing occasionally with a run config on the IDE